src/cls/IPM/Main.cls

```diff
@@ -1863,6 +1888,19 @@ ClassMethod LoadFromRepo(tDirectoryName, ByRef tParams) [ Internal ]
 }
 
 ClassMethod Load(ByRef pCommandInfo) [ Internal ]
+{
+	Set devMode = $Get(pCommandInfo("data", "DeveloperMode"), 1)
+	Set log = ##class(%IPM.General.History).LoadInit()
+	Try {
+		Do ..LoadInternal(.pCommandInfo, log)
+	} Catch Ex {
+		$$$ThrowOnError(log.Finalize(Ex.AsStatus(), devMode))
```

Comment:
General question: if the module install fails and transactions are enabled, does rolling back of the transaction result in the history data getting rolled back as well?